ArcGISRuntimeSDKQt_QMLSamples/Maps/ApplyScheduledMapUpdates/ApplyScheduledMapUpdates.qml

```diff
@@ -133,7 +133,7 @@ Rectangle {
                 syncJob = offlineMapSyncTask.syncOfflineMap(createDefaultOfflineMapSyncParametersResult);
 
                 // connect to the job signals
-                syncJob.jobStatusChanged.connect(()=> {
+                syncJob.statusChanged.connect(()=> {
```

Comment:
i think you should be able to get the status as an arg to this connection, rather than having to poll the `syncJob` again